ui/pages/bridge/transaction-details/transaction-details.tsx

```diff
@@ -60,6 +63,34 @@ const getBlockExplorerUrl = (
   return `${rootUrl}/tx/${txHash}`;
 };
 
+const getBridgeAmount = ({
+  bridgeHistoryItem,
+  srcChainTxMeta,
+}: {
+  bridgeHistoryItem?: BridgeHistoryItem;
+  srcChainTxMeta?: TransactionMeta;
+}) => {
+  if (bridgeHistoryItem) {
+    return `${calcTokenAmount(
+      bridgeHistoryItem.quote.srcTokenAmount,
```

Comment:
This won't include the MM fee (part of the actual transferred amount)

If sentAmount and other [calculated quote metadata](https://github.com/MetaMask/metamask-extension/blob/develop/ui/ducks/bridge/selectors.ts#L233) fields are saved to the txHistory `pricingData` field, we can use `sentAmount` here instead. We should audit other usages of `srcTokenAmount` as well and use `sentAmount` too

Reply:
> Should probably include at least sentAmount in the txMeta as well

Confirmations team's recommendation was to store very bridge specific data in its own controller, so I think that putting it into `txHistory.pricingData` should be sufficient.

Reply:
Done in [b52bf65](https://github.com/MetaMask/metamask-extension/pull/28460/commits/b52bf650be57cd4224149f178fe7cb6f15a2a273) and https://github.com/MetaMask/metamask-extension/pull/28460/commits/110c473a43a69ce99b4bdf770a9a615de52c010b